### ISSUE TITLE
startup message could mention apropos or `?"..."` for text search

### ISSUE THREAD

**Issue by tpapp** [JuliaLang/julia#37523]
Recent discussions on Discourse suggest that not all users know about `apropos` (or equivalently, using `?"..."` for text search). The startup message currently mentions

```
Type "?" for help, "]?" for Pkg help.
```

I suggest that text search is added to this, but I am still looking for a compact way of phrasing it.

**Comment by fredrikekre** [comment]
It could be added to the the help help:
```
help?> help

  Welcome to Julia 1.5.1. The full manual is available at

  https://docs.julialang.org

  as well as many great tutorials and learning resources:

  https://julialang.org/learning/

  For help on a specific function or macro, type ? followed by its name, e.g. ?cos, or ?@time, and press enter. Type ; to enter shell mode, ] to enter package mode
```

**Comment by KristofferC** [comment]
No one reads that though.

**Comment by tpapp** [comment]
What about adding to `?help`, and also
```
Type ? for help, ?"..." for text search,
?help for more options, ]? for Pkg help.
```
Quotes removed as nesting them would be confusing.

**Comment by apparluk** [comment]
Would a man 7 Miscellanea page suffice?

**Comment by tpapp** [comment]
I don't think that a lot of Julia users would look at `man` pages, as typically Julia docs are not there.

**Comment by apparluk** [comment]
[Getting Started](https://docs.julialang.org/en/v1/manual/getting-started/#man-getting-started) and [julia.1](https://github.com/JuliaLang/julia/blob/master/doc/man/julia.1) share some similar content.

By the way `-q` and `--quiet` still work after the man page modification.

**Comment by apparluk** [comment]
The Getting Started truncated and modified julia(1) content should be replaced with the following.

See also: julia(1)

If a man 7 Miscellanea page is written then replaced with the following.

See also: julia(1), julia(7)

If man 3 C Library Functions page (for `@printf`) is written then replaced with the following.

See also: julia(1), julia(3), julia(7)

**Comment by apparluk** [comment]
The suggestions  https://github.com/JuliaLang/julia/issues/37523#issuecomment-690947641 and https://github.com/JuliaLang/julia/pull/37851 are unaesthetic.

If you want more text add `--banner={yes|no|alt|auto}`, and get rid of the ASCII art in `alt`.

<hr>

If additional man pages are made keep them in ASCII.

I've noticed U+027E8 and U+027E9 in man pages can result in incorrect rendering in UXTerm.

<hr>

**Comment by mcabbott** [comment]
> Type ? for help, ?"..." for text search,
> ?help for more options, ]? for Pkg help.

Typing `?` and typing `?help` seem to produce identical results, mentioning both seems confusing. 

The page they bring up has `type ? followed by its name, e.g. ?cos, or ?@time` which seems clearer than the rather cryptic `?"..."` to me. Although perhaps it should mention `apropos("and")` as a way to search (with a better example!)

**Comment by apparluk** [comment]
`--banner=alt`

![banner](https://user-images.githubusercontent.com/52227507/95139537-fd273680-076c-11eb-906e-88770e71027b.png)

Additional lines of text can go under the two with the URLs.

**Comment by ashwani-rathee** [comment]
@mcabbott @tpapp @apparluk can I take this issue and work on it??

**Comment by tpapp** [comment]
I think that #37851 already fixes this, just needs to be merged.

**Comment by apparluk** [comment]
@ashwani-rathee

I'm not an organization Contributor.

Suggestion for a julia(3) page passed by here: https://github.com/JuliaLang/julia/issues/37466#issuecomment-700383017.

**Comment by apparluk** [comment]
@ashwani-rathee

_cf_: https://github.com/JuliaLang/julia/issues/37646#issuecomment-695195370

I don't know if this ticket interests you or of the opinion of Organization members on the clean up.

**Comment by ashwani-rathee** [comment]
Thanks @apparluk for directing me,well actually it was for hacktoberfest.....but why not be in good books of the people and work that interest me :) ?I think I'll check the #37646 and then move on to more specific packages that interest me like juliaimages and dsp

**Comment by apparluk** [comment]
@ashwani-rathee don't bother to check https://github.com/JuliaLang/julia/issues/37646#issuecomment-695195370.

It's unclear why it was deemed a relevant example to demonstrate printing `\n` in a REPL.

**Comment by baggepinnen** [comment]
The docstring for `apropos` could mention the shorthand `help?> "string"` version as well, I use `apropos` all the time but just learned about this shorthand <3
